# This using sectoid/Feelings instead of defy as the song for the credits. I'd
# like to ignore errors / disable music there, since we can't guarantee that
# even Feelings will be available, but I can't figure out how to catch the
# exception this bit of code is throwing.
Index: fretsonfire-1.3.110.dfsg/src/Credits.py
===================================================================
--- fretsonfire-1.3.110.dfsg.orig/src/Credits.py	2009-05-02 17:35:45.272567223 +0100
+++ fretsonfire-1.3.110.dfsg/src/Credits.py	2009-05-02 19:05:44.064542527 +0100
@@ -91,8 +91,8 @@
     self.engine      = engine
     self.time        = 0.0
     self.offset      = 1.0
-    self.songLoader  = self.engine.resource.load(self, "song", lambda: Song.loadSong(self.engine, "defy", playbackOnly = True),
-                                                 onLoad = self.songLoaded)
+    self.songLoader  = self.engine.resource.load(self, "song", lambda: Song.loadSong(self.engine, "sectoid/Feelings", playbackOnly = True),
+                                                    onLoad = self.songLoaded)
     self.engine.loadSvgDrawing(self, "background1", "editor.svg")
     self.engine.loadSvgDrawing(self, "background2", "keyboard.svg")
     self.engine.loadSvgDrawing(self, "background3", "cassette.svg")
